dftimewolf/lib/collectors/timesketch.py

```diff
@@ -178,33 +181,52 @@ def _GetSketch(
     if not timesketch_api.session:
       self.ModuleError('Could not connect to Timesketch server.', critical=True)
 
-    sketch_obj = timesketch_api.get_sketch(self.sketch_id)
+    sketch_obj = timesketch_api.get_sketch(sketch_id)
     if not sketch_obj:
-      self.ModuleError(f'Could not get sketch {self.sketch_id}', critical=True)
+      self.ModuleError(f'Could not get sketch {sketch_id}', critical=True)
     self.state.AddToCache('timesketch_sketch', sketch_obj)
     return sketch_obj
 
-  def _GetSearchResults(self) -> pd.DataFrame:
+  def _GetSearchResults(
+    self,
+    selected_sketch: sketch.Sketch,
+    query_string: str,
+    return_fields: str,
+    start_datetime: datetime.datetime | None = None,
+    end_datetime: datetime.datetime | None = None,
+    labels: list[str] | None = None,
+    indices: list[int] | None = None,
```

Comment:
style nit: add 2 spaces for hanging indent

Reply:
Done! Autoformatter ftw.